spyder/plugins/ipythonconsole/widgets/main_widget.py

```diff
@@ -1777,8 +1777,8 @@ def create_new_client(self, give_focus=True, filename='', special=None,
         return client
 
     def create_client_for_kernel(self, connection_file, hostname, sshkey,
```

Comment:
```suggestion
    def create_client_for_kernel(
        self,
        connection_file,
        hostname,
        sshkey,
```

Improve code formatting.